data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key.status/UnmanagedKeyStatus.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "UnmanagedKeyStatus",
+  "namespace": "net.corda.data.crypto.wire.ops.key.status",
+  "doc": "Defines the key status and key rotation status data stored in the database.",
```

Comment:
I wouldn't use the word database here as there are future plans to sit the statemanager on other things.

Reply:
Good point. Removed.